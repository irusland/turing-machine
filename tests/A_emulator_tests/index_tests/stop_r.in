2
delete stop
delete stop
3
delete 1 -> delete _ R
delete 0 -> delete 0 R
delete _ -> stop _ R
110011