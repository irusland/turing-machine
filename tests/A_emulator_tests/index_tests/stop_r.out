STOP after 7 transitions
00
stop 5
